.github/workflows/reusable-contract.yml

```
@@ -0,0 +1,47 @@
+
+      - name: 🧪 Run ID tests
+        id: id-contract-test
+        run: bun test id.test.ts
+
+      - name: 🧪 Run POST tests
+        id: post-contract-test
+        run: bun test post.test.ts
+
+      - name: 📤 Set result output
+        id: set-result
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:73: Double quote to prevent globbing and word splitting

(shellcheck)